packages/shortcuts-extension/style/base.css

```
@@ -0,0 +1,394 @@
+/*-----------------------------------------------------------------------------
+| Copyright (c) Jupyter Development Team.
+| Distributed under the terms of the Modified BSD License.
+|----------------------------------------------------------------------------*/
+
+/* Shortcut Input Style */
+
+.jp-Shortcuts-InputBox {
+  display: inline-flex;
+  padding-top: 2px;
+}
+
+.jp-Shortcuts-InputBoxNew {
+  margin-left: 10px;
+}
+
+.jp-mod-hidden {
+  display: hidden;
+}
+
+@keyframes slide-animation {
+  from {
+    width: 0;
+    left: 0;
+  }
+
+  to {
+    width: 120px;
+    left: 0;
+  }
+}
+
+.jp-Shortcuts-Input {
+  animation-duration: 0.5 s;
+  animation-timing-function: ease-out;
+  animation-name: 'slide-animation';
+  border-width: var(--jp-border-width);
+  border-color: var(--jp-border-color3);
+  border-style: solid;
+  background-color: var(--jp-layout-color0);
+  margin-left: auto;
+  padding-left: 10px;
+  width: 120px;
+  height: 25px;
+  line-height: 25px;
+  display: block;
+}
+
+.jp-Shortcuts-Input:focus {
+  outline: none;
+  color: var(--jp-content-font-color1);
+  border-color: var(--jp-brand-color2);
+}
+
+.jp-mod-unavailable-Input:focus {
+  border-color: var(--jp-error-color2);
+}
+
+.jp-Shortcuts-InputText {
+  overflow-x: hidden;
+  overflow-y: hidden;
+  margin: 0;
+  margin-top: 4px;
+  padding: 0 5px;
+  height: 17px;
+  line-height: 17px;
+  width: fit-content;
+}
+
+.jp-mod-selected-InputText {
+  background-color: var(--jp-brand-color3);
+  overflow: hidden;
+}
+
+.jp-mod-waiting-InputText {
+  color: var(--jp-content-font-color3);
+}
+
+.jp-Shortcuts-Submit {
+  background-color: var(--jp-brand-color2);
+  border-radius: 0;
+  border: none;
+  color: var(--jp-layout-color0);
+  font-family: var(--jp-ui-font-family);
+  display: block;
+  height: 27px;
+  width: 26px;
+  cursor: pointer;
+}
+
+.jp-Shortcuts-Submit:focus {
+  outline: none;
+}
+
+.jp-Shortcuts-Submit .jp-icon3[fill] {
+  fill: var(--jp-layout-color1);
+}
+
+.jp-Shortcuts-Submit.jp-mod-defunc-Submit {
+  background-color: var(--jp-layout-color3);
+}
+
+.jp-Shortcuts-Submit.jp-mod-defunc-Submit .jp-icon3[fill] {
+  fill: var(--jp-inverse-layout-color1);
+}
+
+.jp-Shortcuts-Submit.jp-mod-conflict-Submit {
+  background-color: var(--jp-error-color1);
+}
+
+/* Shortcut Item Style */
+.jp-Shortcuts-Cell {
+  padding: 6px 12px;
+  display: table-cell;
+  width: 20%;
+  vertical-align: middle;
+}
+
+.jp-Shortcuts-ShortcutCell {
+  display: flex;
+  min-width: 100px;
+  flex-wrap: wrap;
+}
+
+.jp-Shortcuts-EmptyCell {
+  height: 32px;
+}
+
+.jp-Shortcuts-Row {
+  padding: 10px;
+  width: 100%;
+  display: table-row;
+  border-bottom: var(--jp-border-width) solid var(--jp-border-color1);
+  vertical-align: middle;
+  background-color: var(--jp-layout-color0);
+}
+
+.jp-Shortcuts-Row:hover #shortcut-keys {
```

```suggestion
.jp-Shortcuts-Row:hover .jp-Shortcuts-ShortcutKeys {
```

Replace ID with class name